src/applications/personalization/profile/tests/fixtures/communication-preferences/get-200-maximal.json

```diff
@@ -48,15 +48,15 @@
           "description": null,
           "communicationItems": [
             {
-              "id": 2,
-              "name": "COVID-19 Updates",
+              "id": 11,
+              "name": "Biweekly MHV newsletter",
               "communicationChannels": [
                 {
                   "id": 2,
                   "name": "Email",
                   "description": "Email Notification",
                   "communicationPermission": {
-                    "id": 4281,
```

Comment:
Turns out this Group: 2 at one point was active and was used for COVID updates, so now we are just mocking it for the upcoming newsletter pref.